packages/@o3r/workspace/schematics/ng-add/helpers/linter.ts

```diff
@@ -0,0 +1,25 @@
+import { SchematicContext } from '@angular-devkit/schematics';
+import { askConfirmation } from '@angular/cli/src/utilities/prompt';
+
+/**
+ * Checks if `eslint` package is installed. If so, ask the user for otter linter rules install.
+ * Otherwise displays a message to inform the user that otter linter rules can be added later.
+ *
+ * @param context Schematics context
+ */
+export const shouldOtterLinterBeInstalled = async (context: SchematicContext): Promise<boolean> => {
+  const linterPackageName = 'eslint';
+  let useOtterLinter = false;
+  try {
+    require.resolve(`${linterPackageName}/package.json`);
+    if (context.interactive) {
+      useOtterLinter = await askConfirmation(`You already have eslint installed. Would you like to add otter config rules for eslint ?
```

Comment:
nitpicking: in english there is no space before `?` or `!` :)
```suggestion
      useOtterLinter = await askConfirmation(`You already have eslint installed. Would you like to add Otter config rules for eslint?
```

Reply:
Code copy pasted from `@o3r/core`